rules/misc_rules.build_defs

```diff
@@ -286,16 +286,16 @@ def hash_filegroup(name:str, srcs:list=None, deps:list=None, exported_deps:list=
     return build_rule(
         name=name,
         srcs=srcs,
-        cmd='',
+        output_dirs = ['_out'],
+        cmd = 'mkdir _out; for S in $SRCS; do B="$(basename $S)"; mv "$S" "_out/${B%.*}-$(shasum -a 256 $S | cut -d " " -f 1 ).${B##*.}"; done',
```

Comment:
I have no idea what this does but I trust you that it works.

Reply:
It turns a filename like `script.js` into `script-a13ffe6ee73459f04e180132a3e38c3e5fc145e27b4a5a4613960b01058680aa.js`